.github/workflows/ci.yml

```diff
@@ -5,71 +5,107 @@ on:
   push:
     branches: [ master, develop ]
   pull_request:
-    branches: [ master, develop ]
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
 jobs:
-  nanos-build:
+  nano_build:
+    name: Build Hedera application for NanoS, X and S+
+    strategy:
+      matrix:
+        include:
+          - SDK: "$NANOS_SDK"
+            device: nanos
+          - SDK: "$NANOX_SDK"
+            device: nanox
+          - SDK: "$NANOSP_SDK"
+            device: nanosp
     runs-on: ubuntu-latest
     container:
-      image: ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest
+      image: ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder-lite:latest
+
     steps:
-      - uses: actions/checkout@v2
-      - name: Build app
+      - name: Clone
+        uses: actions/checkout@v3
+
+      - name: Build
         run: |
-          PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python make clean
-          PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python make
+          git config --global --add safe.directory "$GITHUB_WORKSPACE"
+          PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python make BOLOS_SDK=${{ matrix.SDK }}
```

Comment:
Can't this `PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION` env variable be set in the Makefile only?
So that anyone wanted to build the app doesn't need to do this?
else we should update the README.

Reply:
it's a temporary workaround

Reply:
I've modified the readme